spec/homepage_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'pg'
+require 'peep'
+
+describe '.all' do
+  it 'returns all peeps in reverse chronological order' do
+    connection = PG.connect(dbname: 'chitter_test')
+
+    # Add the test data
+    Peep.create('New Peep 1!')
+    Peep.create('Yoyoyo!')
+    Peep.create('http://www.google.com')
+
+    peeps = Peep.all
+
+    expect(peeps).to include('New Peep 1!')
+    expect(peeps).to include('Yoyoyo!')
+    expect(peeps).to include('http://www.google.com')
```

Comment:
As you're using `include`, this doesn't actually test that the peeps are in the right order.